homework.txt

```diff
@@ -0,0 +1,186 @@
+       problem 1
+                         //Multiplication table..
+
+// iterate from 0 to 10
+for(var i=0;i<10;i++){
+    var result=i*9; //for each iterate Multiply the number by 9
+    console.log(result); //log the result..
+}// end for
+
+                                             //>>>>>>>>Bonus<<<<<<<<
+
+for(var i=1;i<=10;i++){
+   console.log("*** Multiply by "+i+" ***"); //which table I am in (table i)
+            // used nested loop to show the table for every multiplier
+   for(var j=1;j<=10;j++){
+      var result= i*j; //for each iterate Multiply the number from 1 to 10
+      console.log(i+" * "+j+" = "+result+""); //log the result..
+    } //end for j
+} //end for i
```

Comment:
Awesome! We can also use `for(var j=i ..)` to get rid of duplicates, since `3*4 === 4*3` so we don't need to do `4*3` if we have done `3*4`. Hope this makes sense.